src/NServiceBus.Core/Timeout/Hosting/Windows/TimeoutPersisterReceiver.cs

```diff
@@ -19,6 +19,16 @@ class TimeoutPersisterReceiver : IDisposable
         public CriticalError CriticalError { get; set; }
         public Address DispatcherAddress { get; set; }
         public TimeSpan TimeToWaitBeforeTriggeringCriticalError { get; set; }
+        internal DateTime NextRetrieval { get; private set; } = DateTime.UtcNow;
```

Comment:
This went from a field initializer to an auto-property initializer, so nothing has changed about the timing.

> The initializers are executed in order as written, just as – and along with – field initializers.